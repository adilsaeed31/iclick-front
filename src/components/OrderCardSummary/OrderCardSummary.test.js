import React from "react";
import renderer from "react-test-renderer";
import OrderCardSummary from "./OrderCardSummary";

const summary = {
  fulfillmentTotal: { amount: 2.99, displayAmount: "$2.99" },
  itemTotal: { amount: 6000, displayAmount: "$6,000.00" },
  surchargeTotal: { amount: 0, displayAmount: "$0.00" },
  taxTotal: { amount: 0, displayAmount: "$0.00" },
  total: { amount: 6002.99, displayAmount: "$6,002.99" }
};

test("basic snapshot of an order summary", () => {
  const component = renderer.create((
    <OrderCardSummary
      summary={summary}
    />
  ));

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
