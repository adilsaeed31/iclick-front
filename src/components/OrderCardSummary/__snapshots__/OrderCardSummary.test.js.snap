// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic snapshot of an order summary 1`] = `
.c0 {
  width: 100%;
  border-spacing: 0;
  background-color: transparent;
  padding-left: 0;
  padding-right: 0;
  padding-top: 0;
  padding-bottom: 0;
}

.c1 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  border-top-color: #e6e6e6;
  border-top-style: solid;
  border-top-width: 0;
  padding-bottom: 8px;
  padding-left: 0;
  padding-right: 0;
  padding-top: 8px;
  text-align: left;
}

.c7 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  border-top-color: #e6e6e6;
  border-top-style: solid;
  border-top-width: 1px;
  padding-bottom: 8px;
  padding-left: 0;
  padding-right: 0;
  padding-top: 8px;
  text-align: left;
}

.c2 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  text-align: right;
}

.c3 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  border-top-color: #e6e6e6;
  border-top-style: solid;
  border-top-width: 0;
  padding-bottom: 16px;
  padding-left: 0;
  padding-right: 0;
  padding-top: 8px;
  text-align: left;
}

.c4 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  padding-bottom: 16px;
  text-align: right;
}

.c5 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 14px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 400;
  -webkit-letter-spacing: .02em;
  -moz-letter-spacing: .02em;
  -ms-letter-spacing: .02em;
  letter-spacing: .02em;
  line-height: 1;
  border-top-color: #e6e6e6;
  border-top-style: solid;
  border-top-width: 1px;
  padding-bottom: 16px;
  padding-left: 0;
  padding-right: 0;
  padding-top: 16px;
  text-align: left;
}

.c6 {
  text-align: right;
  padding-bottom: 16px;
  padding-top: 16px;
}

.c8 {
  -webkit-font-smoothing: antialiased;
  color: #3c3c3c;
  font-family: 'Source Sans Pro','Helvetica Neue',Helvetica,sans-serif;
  font-size: 18px;
  font-style: normal;
  font-stretch: normal;
  font-weight: 700;
  -webkit-letter-spacing: .03em;
  -moz-letter-spacing: .03em;
  -ms-letter-spacing: .03em;
  letter-spacing: .03em;
  line-height: 1;
}

<table
  className="c0"
>
  <tbody>
    <tr>
      <td
        className="c1"
      >
        Items:
      </td>
      <td
        className="c2 c1"
      >
        $6,000.00
      </td>
    </tr>
    <tr>
      <td
        className="c1"
      >
        Shipping:
      </td>
      <td
        className="c2 c1"
      >
        $2.99
      </td>
    </tr>
    <tr>
      <td
        className="c1"
      >
        Surcharges:
      </td>
      <td
        className="c2 c1"
      >
        $0.00
      </td>
    </tr>
    <tr>
      <td
        className="c3"
      >
        Tax:
      </td>
      <td
        className="c4 c1"
      >
        $0.00
      </td>
    </tr>
    <tr>
      <td
        className="c5"
      >
        Order total:
      </td>
      <td
        className="c6 c7"
      >
        <span
          className="c8"
        >
          $6,002.99
        </span>
      </td>
    </tr>
  </tbody>
</table>
`;
